car_directory/dir.go

```diff
@@ -53,16 +53,25 @@ func (c *Car) Render(out chan<- string) {
 	// Calculate max directory elements to display.
 	maxLength := 3
 	if e := os.Getenv("BULLETTRAIN_CAR_DIRECTORY_MAX_LENGHT"); e != "" {
-		if ml, err := strconv.Atoi(e); err == nil && ml >= 3 {
+		if ml, err := strconv.Atoi(e); err == nil && ml >= 0 {
 			maxLength = ml
 		}
 	}
 
 	// Compose directory segments.
 	ps := string(os.PathSeparator)
 	dirs := strings.Split(dir, ps)
-	if maxLength > 0 && len(dirs) > maxLength+1 {
-		newPath := make([]string, 0)
+	newPath := make([]string, 0)
```

Comment:
`newPath` should be moved down to the next `else` block where it is really used.

Reply:
See my comment bellow.